src/app/backend-api.service.ts

```diff
@@ -698,4 +742,42 @@ export class BackendAPIService {
       TxnHashHex,
     });
   }
+
+  AuthorizeDerivedKey(
+    OwnerPublicKeyBase58Check: string,
+    DerivedPublicKeyBase58Check: string,
+    ExpirationBlock: number,
+    AccessSignature: string,
+    TransactionSpendingLimitHex: string
+  ): Observable<any> {
+    const req = this.post('authorize-derived-key', {
+      OwnerPublicKeyBase58Check,
+      DerivedPublicKeyBase58Check,
+      DerivedKeySignature: false,
+      ExpirationBlock,
+      MinFeeRateNanosPerKB: 1000,
```

Comment:
not an issue for now, but we'll need to update this if the network min fee rate is ever updated.